src/devices/orvibo.js

```diff
@@ -56,6 +84,15 @@ module.exports = [
             'button_3_click', 'button_3_hold', 'button_3_release', 'button_4_click', 'button_4_hold', 'button_4_release'])],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['RemoteControl-22'],
+        model: 'RC808ZB',
+        vendor: 'ORVIBO',
+        description: 'ORVIBO Remote Control',
+        fromZigbee: [fzLocal.orvibo_raw_3, fz.battery],
+        exposes: [e.battery()],
```

Comment:
I notice that `e.action([])` is missing here, please add all the possible actions here.

Example: https://github.com/Koenkk/zigbee-herdsman-converters/pull/5786/files#diff-d81231be627ae3c846eecc8731c4130661783a89b5d05f855658e9755d1b994bR83